src/components/navbar.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const Navbar = () => (
```

Comment:
- [ ] In the Navbar component, please add the icon displayed at the very right-top corner of the header as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏